### PR TITLE
Devx 2299 tutorials

### PR DESCRIPTION
**Pull request by fabianrbz**
## Description

Change the way we write tutorials.
Now, every tutorial must have a folder under `config/tutorials/en` with a file called `base.yml` that acts as entry point, with the option to define programming language specific files.

### CODE REVIEW

File: config/landing_pages/hansel.yml [hunk 1]
```diff
@@ -76,12 +76,12 @@ page:
       column:
       - type: tutorial
         tutorial:
-          name: two-factor-authentication-dotnet-verify-api
+          name: two-factor-authentication-dotnet-verify-api/base
```
Comment (mheap) [inline config/landing_pages/hansel.yml:79]: Could we strip the `/base` off this in the URL?

Comment (fabianrbz) [reply config/landing_pages/hansel.yml:79]: that's not an url, it's the name of the tutorial.
It's possible, the problem is that sometimes you might want to use a language specific tutorial, we could check if the name has a programming language, and if not, use the default one.

Comment (fabianrbz) [reply config/landing_pages/hansel.yml:79]: this is applicable only to landing pages though.

**Comment by mheap** [comment]
@fabianrbz Is this ready for review?

**Comment by fabianrbz** [comment]
@mheap not yet

**Comment by bencgreenberg** [comment]
@fabianrbz Do we still need this PR open?

**Pull request closed without merging**